crates/swc_common/src/syntax_pos.rs

```diff
@@ -86,6 +88,7 @@ impl Globals {
         Globals {
             hygiene_data: Mutex::new(hygiene::HygieneData::new()),
             dummy_cnt: AtomicU32::new(DUMMY_RESERVE),
+            mark_count: AtomicU32::new(1),
```

Comment:
Using `AtomicU32`,  because `mark_count`(from original `HygineData.marks`) is only used in `Mark::fresh`, the `HygineData.marks` is used exclusive with `HygineData.syntax_contexts` and `HygineData.markings`, splitting them could improve concurrency in parallel scenario (`Mark::fresh` will not lock anymore, `Atomic` is faster than Mutex, even `parking_lot`), you could see the benchmark result.